images/structure.md

```diff
@@ -43,3 +43,29 @@ USER coder
 Your users can then create descendant images. These contain all of the original
 tooling and configuration installed onto the base image and the new
 customizations added by the users.
+
+## Coder assets
+
+Coder inserts static assets into each workspace, including code-server, JetBrains
+IDEs, and the Coder CLI. These assets are installed into the `/var/tmp/coder` directory
```

Comment:
We don't inject jetbrains IDEs. I think you're confusing this with the code-server jetbrains plugin, which we don't need to document publicly.

Reply:
The prior wording was "assets that enable" the JetBrains IDEs, so I'll make the change to explicitly reference the Projector binary. Thanks for catching this.